### ISSUE TITLE
Test Home Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4898]
# Page to Test:

Home  - https://openlibrary.org/

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly I am working on this.

**Comment by ArunTeltia** [comment]
You can have a look at this @Yashs911 , I tested this page,
I dont find any problem with this I tested the accessibility of the homepage by going through each and every element of the page by going from top to bottom and bottom to top I dont found any issues, Only the navbar issues are there which i have created a new issue separately

**Comment by Yashs911** [comment]
@bpmcneilly 

1. Due to `opacity: 0;` used for hiding the selected option on select dropdown one can't see the outline and hence can't use the spacebar to open `select` dropdown.
![image](https://user-images.githubusercontent.com/64412143/113246977-ac396a80-92d7-11eb-8316-e60a77d3a5d0.png)
![image](https://user-images.githubusercontent.com/64412143/113247013-bbb8b380-92d7-11eb-98b7-84329f35d7db.png)
